act/signed-commit/action.yml

```diff
@@ -29,30 +23,77 @@ inputs:
     required: false
     default: "."
 
-  github-token:
-    description: >-
-      Github token used for interacting with Actions API. Must have Contents:Write permissions
-    required: true
+outputs:
+  commit-sha:
+    description: SHA of the created commit
```

Comment:
To make these outputs functional I think you'll need to specify a `value:` field here with a reference to the step output that should populate them. This is to "move" the output from the step scope to the action scope. An example of this is [here](https://github.com/freedomofpress/actionslib/blob/main/act/delete-artifact/action.yml#L30)